examples/plot_scalable_poly_kernels.py

```diff
@@ -0,0 +1,118 @@
+
+data = fetch_covtype()
+X, y = data["data"], data["target"]
+
+y[y != 2] = 0
+y[y == 2] = 1  # We will try to separate class 2 from the other 6 classes.
+
+##############################################################################
+# Here we select 20,000 samples for training and 20,000 for testing.
+# To actually reproduce the results in the original Tensor Sketch paper,
+# select 100,000.
+
+X_train, X_test, y_train, y_test = train_test_split(X, y, train_size=20_000,
+                                                    test_size=20000,
+                                                    random_state=42)
+
+##############################################################################
+# Now scale features to the range [0, 1] to match the format of the dataset in
+# the LIBSVM webpage, and then normalize to unit length as done in the
+# original Tensor Sketch paper [1].
+
+mm = MinMaxScaler().fit(X_train)
+X_train = normalize(mm.transform(X_train))
+X_test = normalize(mm.transform(X_test))
+
+##############################################################################
+# As a baseline, train a linear SVM on the original features and print the
+# accuracy:
+
+lsvm = LinearSVC()
+lsvm.fit(X_train, y_train)
+lsvm_score = 100*lsvm.score(X_test, y_test)
+print(f"Linear SVM score on raw features: {lsvm_score:.2f} %%")
+
+##############################################################################
+# Then we train linear SVMs on the features generated by
+# :class:`PolynomialSampler` with different values for `n_components`, showing
+# that these kernel feature approximations improve the accuracy of linear
+# classification:
+
+for n_components in [200, 500, 1000, 1500]:
+
+    pipeline = Pipeline(steps=[("kernel_approximator",
+                                PolynomialSampler(
+                                    n_components=n_components,
+                                    degree=4)),
+                               ("linear_classifier", LinearSVC())])
+    pipeline.fit(X_train, y_train)
+    ps_lsvm_score = 100 * pipeline.score(X_test, y_test)
+
+    print(f"Linear SVM score on {n_components} PolynomialSampler " +
+          f"features: {ps_lsvm_score:.2f} %%")
+
+##############################################################################
+# Finally, train a kernelized SVM to see how well :class:`PolynomialSampler`
+# is approximating the performance of the kernel. This of course may take some
+# time, as the SVC class has a relatively poor scalability. This is the
+# reason why kernel approximators are so useful:
+
+from sklearn.svm import SVC
+ksvm = SVC(C=500., kernel="poly", degree=4, coef0=0,
+           gamma=1.).fit(X_train, y_train)
+ksvm_score = 100*ksvm.score(X_test, y_test)
+print(f"Kernel-SVM score on raw featrues: {ksvm_score:.2f} %%")
+
+##############################################################################
+# As we can see, this kernelized SVM achieves a slightly higher accuracy, but
```

Comment:
I would be in favor of having a visual plot in this example, instead of having only printed accuracy.
I would picture for example an "accuracy versus time" plot, with a dot for each model.

Reply:
Okay, I modified the example a bit to generate the plot you suggested:
![image](https://user-images.githubusercontent.com/7440734/78471878-c5dfba80-7734-11ea-8179-0763b39feaae.png)

Of course the benefit of `PolynomialSampler` becomes more apparent when you have `100,000` training samples instead of `5,000`:

![image](https://user-images.githubusercontent.com/7440734/78471896-eb6cc400-7734-11ea-866d-e7f1aeef80d4.png)

But I think that is fine.